### PR TITLE
Add Digital Collections mapping rules

### PR DESCRIPTION
**Pull request by mlhale7**
## What does this Pull Request do?

This PR adds the mapping for Digital Collections for Primo VE as well as additional sample records.

## What's new?

This PR shares the normalization rules and regex needed to share our Digital Collections with Primo VE. The mapping reflects what is currently in production on Alma. Note that there is a hope to improve the creator and contributor mapping (currently all names are mapped to dc:creator) and that a Salesforce ticket has been opened for this. Given the fact that the current normalization rules have not changed in over a month, I felt it was important to go ahead and share what is presently being used instead of waiting any longer.

## How should this be tested?

Check to make sure the formatting is correct and the readthedocs site will build. 

## Interested parties

@markpbaggett

### CODE REVIEW

File: docs/source/external_sources/digital_collections.rst [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+    	when
+         	exist "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='typeOfResource'][@collection='yes']"   
+    	then
+        	set "digital_collection" in "discovery"."resourceType"
+	end
+	
+RegEx for Links
+===============
+
+Thumbnail: ^(http(s):\/\/).*(TN\/view)$
```
Comment (markpbaggett) [inline docs/source/external_sources/digital_collections.rst]: Just for legibility, let's do an anonymous code block:

```
Thumbnail:

.. code-block::

    ^(http(s):\/\/).*(TN\/view)$

```

File: docs/source/external_sources/digital_collections.rst [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+
+Object (Item/Collection): ^(http(s):\/\/digital\.lib\.utk\.edu\/collections\/islandora\/object\/).*([0-9]|collections%3A.*|gsmrc%3A.*|arrowmont%3A.*)$
```
Comment (markpbaggett) [inline docs/source/external_sources/digital_collections.rst]: Same here:

```
Object (Item/Collection): 

.. code-block::

    ^(http(s):\/\/digital\.lib\.utk\.edu\/collections\/islandora\/object\/).*([0-9]|collections%3A.*|gsmrc%3A.*|arrowmont%3A.*)$


```

**Review by markpbaggett — CHANGES_REQUESTED** [review]

File: docs/source/external_sources/digital_collections.rst (hunk 2 above)
Comment (mlhale7) [reply docs/source/external_sources/digital_collections.rst]: @markpbaggett - thanks for this suggestion. I've added code blocks for both regular expressions.

**Review by markpbaggett — APPROVED** [review]
:+1:

**Pull request merged**